README.md

```diff
@@ -221,6 +223,119 @@ In the example above, there are three strains: `default` serves content from `ww
+    code: *localRepo
+    content: *localRepo
+    static: *localRepo
+```
+
+The Helix Development server will automatically start a git server that can serve the content from
+the local repository.
+
+### Use the GitHub emulator
+
+When starting the `hlx up` with `--local-repo` argument(s), it instructs the Helix Development Server to
+start a git server that emulates GitHub repositories for a local git repository. All the strains that
+have a _content_ or _static_ url that matches the `origin` of emulated repository are internally reconfigured
+to use the local git server instead.
+
+For the simple case, where only one repository is used for code, content and static just do:
+
+```
+$ hlx up --local-repo
+``` 
+
+#### Multi Strain Example
+
+In the following config, we define 2 repositories:
+
+- `defaultRepo` contains the project's code and the main content
+- `apiRepo` contains additional content; for example the API documentation.
+
+We also define 2 strains, one for each purpose.
+
+```yaml
+definitions:
+  repos:
+    - &defaultRepo https://github.com/helix/welcome.git#master
+    - &apiRepo https://github.com/helix/welcome-api.git#master
+
+strains:
+  - name: api
+    url: https://www.project-helix.io/api
+    code: *defaultRepo
+    content: *apiRepo
+    static: *apiRepo
+
+  - name: default
+    url: https://www.project-helix.io/
+    code: *defaultRepo
+    content: *defaultRepo
+    static: *defaultRepo
+```
+
+Usually, when invoking `hlx up` without any arguments, the Helix Development Server will serve the
+content directly from GitHub. This is not suitable for local development. Also, the `api` strain
+will never be selected, because the `localhost:3000` host header will not match the specified `url` 
+condition.
+
+Starting the server with:  
+
+```
+$ hlx up --host=www.project-helix.io
+```
+
+Solves the latter problem. the `--host` argument internally overrides the `request.header`, so that
+the strain resolution works as desired.
+
+Assume that we also have a local checkout of the `welcome-api`, beside the `welcome` repository:
+
+```
+projects/
+├── welcome/
+│   ├── helix-config.yaml
+│   └── index.md
+└── welcome-api/
+    └── index.md
+```
+
+We can now launch the server with the respective `--local-repo` arguments: 
+
+```
+$ hlx up --host=www.project-helix.io --local-repo=. --local-repo=../welcome-api
+```
+
+Now the server will transiently reconfigure the strains, so that the emulated repositories are used.
+
+> **Note**: If you turn on `--log-level=debug` you should see log entries for the emulated repositories:   
+```
+[hlx] debug: git emulating https://github.com/helix/welcome.git via http://127.0.0.1:52270/helix/github.com--helix--welcome.git#master from './'
+[hlx] debug: git emulating https://github.com/helix/welcome-api.git via http://127.0.0.1:52270/helix/github.com--helix-welcome-api.git#master from '../welcome-api'
+```
+
+For convenience, you can also specify the arguments in an `.env` file:
```

Comment:
Can we explicitly document how to configure the `HLX_LOCAL_REPO` env var? It seems like it must be comma-separated for multiple repos. If that is so, can we document this somewhere here? Perhaps as a comment in the `dotenv` example?

Reply:
see #698